README.md

```diff
@@ -37,91 +34,83 @@ If you'd like to use the accelerator-specific features of 🤗 Optimum, you can
 from transformers import AutoTokenizer
+from optimum.onnxruntime import ORTModelForQuestionAnswering
 
-model_checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
-save_directory = "tmp/onnx/"
-# Load a model from transformers and export it to ONNX
-tokenizer = AutoTokenizer.from_pretrained(model_checkpoint)
-ort_model = ORTModelForSequenceClassification.from_pretrained(model_checkpoint, from_transformers=True)
-# Save the ONNX model and tokenizer
-ort_model.save_pretrained(save_directory)
-tokenizer.save_pretrained(save_directory)
-```
+model_name = "roberta_base_qa"
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+ort_model = ORTModelForQuestionAnswering.from_pretrained(model_name)
 
-Let's see now how we can apply dynamic quantization with ONNX Runtime:
+question = "What's Optimum?"
+text = "Optimum is an awesome library everyone should use!"
+inputs = tokenizer(question, text, return_tensors="pt") 
 
-```python
-from optimum.onnxruntime.configuration import AutoQuantizationConfig
-from optimum.onnxruntime import ORTQuantizer
-
-# Define the quantization methodology
-qconfig = AutoQuantizationConfig.arm64(is_static=False, per_channel=False)
-quantizer = ORTQuantizer.from_pretrained(ort_model)
-# Apply dynamic quantization on the model
-quantizer.quantize(save_dir=save_directory, quantization_config=qconfig)
-```
+# Run with ONNX Runtime.
+outputs = ort_model(**inputs)
 
-In this example, we've quantized a model from the Hugging Face Hub, in the same manner we can quantize a model hosted locally by providing the path to the directory containing the model weights. The result from applying the `quantize()` method is a `model_quantized.onnx` file that can be used to run inference. Here's an example of how to load an ONNX Runtime model and generate predictions with it:
+answer_start_index = outputs.start_logits.argmax()
+answer_end_index = outputs.end_logits.argmax()
 
-```python
-from optimum.onnxruntime import ORTModelForSequenceClassification
-from transformers import pipeline, AutoTokenizer
-
-model = ORTModelForSequenceClassification.from_pretrained(save_directory, file_name="model_quantized.onnx")
-tokenizer = AutoTokenizer.from_pretrained(save_directory)
-classifier = pipeline("text-classification", model=model, tokenizer=tokenizer)
-results = classifier("I love burritos!")
+predict_answer_tokens = inputs.input_ids[0, answer_start_index : answer_end_index + 1]
+answer = tokenizer.decode(predict_answer_tokens, skip_special_tokens=True)
 ```
 
-You can find more examples in the [documentation](https://huggingface.co/docs/optimum/onnxruntime/quickstart) and in the [examples](https://github.com/huggingface/optimum/tree/main/examples/onnxruntime).
+### TensorFlow Lite
+
+Just as for ONNX, it is possible to export to [TensorFlow Lite](https://www.tensorflow.org/lite) and quantize them:
 
+```plain
+optimum-cli export tflite -m deepset/roberta-base-squad2 --sequence_length 384  --quantize int8-dynamic roberta_tflite_model
```

Comment:
Can you break line so that the full command is visible without right scrolling?

Reply:
I tried but then the command does not run by simply copy-pasting, do you have a suggestion?

Reply:
You can do something like:
```
optimum-cli export onnx \
    --model gpt2 \
    gpt2_onnx/
```

Or maybe since it's subprocess under the hood this is failing for tflite?

Reply:
It worked thanks! I might have put spaces after the `\`...